zkmarek/video/slides/episode2/TeX files/operations.tex

```diff
@@ -0,0 +1,6 @@
+\begin{itemize}
```

Comment:
Let's stick to names with underscore_case, not spaces too, ie
Instead of `zkmarek/video/slides/episode2/TeX files/operations.tex`:
`zkmarek/video/slides/episode2/tex/operations.tex`

note files have no meaning here, all folders contains files :)